.github/workflows/markdown-link-checker.yml

```diff
@@ -1,14 +0,0 @@
-name: Check markdown links
```

Comment:
we don't need this CI now ?

Reply:
yes, because Apache Infra doen't allow this one

Reply:
See this one: https://cwiki.apache.org/confluence/display/BUILDS/GitHub+Actions+status#GitHubActionsstatus-Security